README.md

```diff
@@ -1,5 +1,6 @@
 # odin-devices
 [![Build Status](https://travis-ci.com/stfc-aeg/odin-devices.svg?branch=master&service=github)](https://travis-ci.com/stfc-aeg/odin-devices)
 [![Coverage Status](https://coveralls.io/repos/github/stfc-aeg/odin-devices/badge.svg?branch=master&service=github)](https://coveralls.io/github/stfc-aeg/odin-devices?branch=master)
```

Comment:
You can drop the coveralls badge now we're using codecov for the coverage. I think the reporting will kick in properly once this PR is merged to master.